universal-application-tool-0.0.1/app/views/components/FieldWithLabel.java

```diff
@@ -163,6 +169,7 @@ public ContainerTag getContainer() {
     fieldTag
         .withCondId(!Strings.isNullOrEmpty(this.id), this.id)
         .withName(this.fieldName)
+        .condAttr(!this.enabled, "disabled", "true")
```

Comment:
Nit. I'd prefer `this.disabled` but maybe just add a TODO and someone can do a quick follow up.

Reply:
Pushed that up a layer.